docs/getting-started-guides/docker.md

```diff
@@ -37,9 +37,8 @@ Running Kubernetes locally via Docker
 
 - [Overview](#overview)
 - [Prerequisites](#prerequisites)
-- [Step One: Run etcd](#step-one-run-etcd)
-- [Step Two: Run the master](#step-two-run-the-master)
-- [Step Three: Run the service proxy](#step-three-run-the-service-proxy)
+- [Run it](#run-it)
+- [Download kubectl](#download-kubectl-)
```

Comment:
It's required in my local md viewer, but apparently github generates different ids :/

Fixed